### ISSUE TITLE
Fails to install via homebrew on Hight Sierra

### ISSUE THREAD

**Issue by memsom** [jichu4n/prc-tools-remix#2]
I could not manage to get this to get this to compile under Homebrew on High Sierra. I had an issue "undefined reference to libc_name_p." This seems to be related to this [issue on stack overflow](https://stackoverflow.com/questions/3040801/error-compiling-gcc-undefined-reference-to-libc-name-p). To be honest I didn't try to debug the issue, I instead built using the alternate instructions, but had to change the following:

```bash
git clone http://github.com/jichu4n/prc-tools-remix.git
cd prc-tools-remix

mkdir build && cd build
../prc-tools-2.3/configure \
    --enable-targets=m68k-palmos,arm-palmos \
    --enable-languages=c,c++ \
    --disable-nls \
    --with-palmdev-prefix=/usr/local/opt/palmdev \
    --host=i686-apple-darwin
make
sudo make install
```
So, used http protocol (as not set up for ssh on this box) and needed sudo as otherwise the install failed completely. Also worth noting - if you have installed any of the dependencies listed for Linux using Homebrew, the configure script seems to also fail - I was having an issue inititally where binutils was linking __-lfl__ and not __-ll__ and this prevented compilation from succeeding.

Not tested it compiles yet, but m68k-palmos-gcc seems to run with the standard error about no files if you run the command with no params.

**Comment by jichu4n** [comment]
Thanks for the feedback and sorry for the inconvenience! I think this is fixed in PR #1, but I forgot to update the Homebrew formula accordingly. I've [updated](https://github.com/jichu4n/homebrew-palm-os/commit/b47e4b149d733ede569f0a72c1a7137bdf7ef766) the Homebrew formula now, so a `brew update` and then `brew install prc-tools` should work now.

**Comment by gingerbeardman** [comment]
It works! Thanks
